src/System.Runtime/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,5 @@
+#DebuggableAttribute members need to be ported to System.Private.Corelib in .NETNative (https://github.com/dotnet/corefx/issues/9699) :
```

Comment:
I tried that name originally but it doesn’t bind.

The pkg build builds for the following target groups:

   $(TargetGroup) = ‘’
   $(TargetGroup) = ‘net462’

so the decorator has to be one of those.